packages/backend/src/app/authentication/authentication.controller.ts

```diff
@@ -39,6 +46,7 @@ const SignUpRequestOptions = {
 const SignInRequestOptions = {
     config: {
         allowedPrincipals: ALL_PRINICPAL_TYPES,
+        rateLimit: rateLimitOptions,
```

Comment:
Is it a bit extreme to have 2 attempts within 1 minute?

Also, during sign-up, if I input a used, it prompts for another one, or make a wrong attempt login.

I would suggest sign up something low, but sign in I would be more forgiving like 15 per 5 minutes or something like that

Reply:
updated to 50 requests per minute